x/exercise.rb

```diff
@@ -0,0 +1,7 @@
+module X
+  module Exercise
+    def self.exists?(language, slug)
+      Xapi.request('tracks', language, slug).status != 404
```

Comment:
We have a light-weight endpoint: http://x.exercism.io/v3/tracks/:id/exercises/:slug/exists which has a 404 or 200 status